### DISCUSSION TITLE
Configure to step over std functions

### DISCUSSION THREAD

**Discussion by maziac** [microsoft/vscode-cpptools#8535] [Q&A]
In Visual Studio it is possible to configure that a 'step into' into std function (like std::vector::pushBack) is ignored and instead a 'step over' is done.
I.e. with a line like this 
~~~
<Function><Name>std::.*</Name><Action>NoStepInto></Action></Function>
~~~
in default.natstepfilter.

Is there something similar possible with cppvsdbg in vscode?

Stepping into std functions is in most of the cases not wanted and very annoying during debugging.

**Comment by sean-mcmanus** [comment]
I'm not aware of a way. We're tracking a similar issue with https://github.com/microsoft/vscode-cpptools/issues/2488 .

**Reply by maziac** [reply]
Similar, but not the same.
Should I open a bug or feature request?
